### PR TITLE
Spelling mistakes

### PR DESCRIPTION
**Pull request by MacaulayScott**

### CODE REVIEW

**Comment by Philyorkshire** [comment]
All files should be lower case, i.e Index.html -> index.html and so on

File: Index.html [hunk 1]
```diff
@@ -1,14 +1,78 @@
...
 
 <body>
+    <header>
+    <div class="container-nav">
+        <nav>
+            <ul class="float-left">
+             <li><img class="border" src="assets/images/githubsvg.svg" alt="github-logo" height="40px" width="40px"></li> 
+                <li><a href="#">Why GitHub?&nbsp;<span class="rotate-180">^</span></a>
+                    <ul>
+                        <li><a href="#"><b>Features</b></a></li>
```
Comment (brandudno) [inline Index.html]: Can we change the `<b>` tag to `<strong>` please, `<b>` has been deprecated and replaced with `<strong>`

File: Index.html [hunk 2]
```diff
@@ -1,14 +1,78 @@
...
+                        <li><a href="#">Project Management</a></li>
+                        <li><a href="#">Integrations</a></li>
+                        <li><a href="#">Actions</a></li>
+                        <li><a href="#">Packages</a></li>
+                        <li><a href="#">Security</a></li>
+                        <li><a href="#">Team Management</a></li>
+                        <li><a href="#">Hosting</a><hr></li>
+                        <li><a href="#"><b>Customer Stories</b></a></li>
+                        <li><a href="#"><b>Security</b></a></li>
+
```
Comment (brandudno) [inline Index.html]: Please can we remove the whitespace between elements :)

File: Index.html [hunk 3]
```diff
@@ -1,14 +1,78 @@
...
+                        <li><a href="#">Topics</a></li>
+                        <li><a href="#">Collections</a></li>
+                        <li><a href="#">Trending</a></li>
+                        <li><a href="#">Learning Lab</a></li>
+                        <li><a href="#">Open Source Guides</a><hr></li>
+                        <li><a href="#">Connect With Others</a></li>
+                        <li><a href="#">Events</a></li>
+                        <li><a href="#">Community Forum</a></li>
+                        <li><a href="#">GitHub Education</a></li>
+
```
Comment (brandudno) [inline Index.html]: Whitespacing

File: Index.html [hunk 4]
```diff
@@ -1,14 +1,78 @@
...
+                    </ul></a></li>
```
Comment (brandudno) [inline Index.html]: `</a>` I think we have a closing tag here that is never opened

File: Index.html [hunk 5]
```diff
@@ -1,14 +1,78 @@
...
+                <li><a href="#">Pricing&nbsp;<span class="rotate-180">^</span></a>
+                    <ul>
+                        <li><a href="#"><b>Plans</b></a></li>
+                        <li><a href="#">Compare Plans</a></li>
+                        <li><a href="#">Contact Sales</a><hr></li>
+                        <li><a href="#"><b>None Profit</b></a></li>
+                        <li><a href="#"><b>Education</b></a></li>
+
+
+                    </ul></a></li>
```
Comment (brandudno) [inline Index.html]: Same closing tag here

File: Index.html [hunk 6]
```diff
@@ -1,14 +1,78 @@
...
+            </ul>
+            <ul class="float-right">
+                <li>
+
```
Comment (brandudno) [inline Index.html]: Whitespacing

File: assets/css/Styles.css [hunk 7]
```diff
@@ -1,3 +1,7 @@
 body{
     text-align: center;
+}
+
+p{
```
Comment (brandudno) [inline assets/css/Styles.css]: please can include a space between the character and the curly brace so `p {`

File: assets/css/Styles.css [hunk 8]
```diff
@@ -1,3 +1,183 @@
...
\ No newline at end of file
+*{
+    padding: 0px;
+    margin: 0px;
+}
+
+body {
+    background-color: #4c5055;
+    font-family:"Work Sans", sans-serif;
+
```
Comment (brandudno) [inline assets/css/Styles.css]: Whitespacing

File: assets/css/Styles.css [hunk 9]
```diff
@@ -1,3 +1,183 @@
...
+    max-width: 110%;
+    height: 60px;
+    background-color: #24292e;
+    padding-top: 15px;
+    padding-left: 70px;
+    padding-right: 70px;
+}
+
+
+ul>* {
```
Comment (brandudno) [inline assets/css/Styles.css]: Spacing between elements `ul > *`

File: assets/css/Styles.css [hunk 10]
```diff
@@ -1,3 +1,183 @@
...
+
+a {
+    text-decoration: none; 
+    color: white;
+    transition: all 0.25s ease-in-out;
+}
+
+a:hover{
+    color: #656b72;
+
```
Comment (brandudno) [inline assets/css/Styles.css:38]: Whitespacing

File: assets/css/Styles.css [hunk 11]
```diff
@@ -1,3 +1,183 @@
...
+   padding: 20px;
+   float: left;
+   border-radius:10px;
+   transition: all 0.25s ease-in-out;
+   opacity: 0;
+   width: 300px
+}
+
+nav ul li:hover ul{
+    visibility: visible;
```
Comment (brandudno) [inline assets/css/Styles.css:94]: Please can everything be in line

File: assets/css/Styles.css [hunk 12]
```diff
@@ -1,3 +1,183 @@
...
+} 
+
+.rotate-180 {
+    transform: rotate(180deg);
+    float: right;
+}
+
+.button{
+    border-radius: 4px;
+  background-color: #24292e;
```
Comment (brandudno) [inline assets/css/Styles.css]: Inline

File: assets/css/Styles.css [hunk 13]
```diff
@@ -1,3 +1,183 @@
...
+    background-color: #656b72;
+    border-radius: 7px;
+    
+}
+
+.border{
+    border-radius: 90%;
+}
+
+
```
Comment (brandudno) [inline assets/css/Styles.css]: Remove whitespacing at the end of file

File: assets/fonts/Fonts.css [hunk 14]
```diff
@@ -0,0 +1 @@
+dfswdf
```
Comment (brandudno) [inline assets/fonts/Fonts.css]: What's this?

**Review by brandudno — CHANGES_REQUESTED** [review]

File: Index.html [hunk 15]
```diff
@@ -1,7 +1,8 @@
 <!doctype html>
 <html>
+
 <head>
-    
+
```
Comment (brandudno) [inline Index.html]: Whitespacing :)

**Review by brandudno — CHANGES_REQUESTED** [review]

**Review by brandudno — APPROVED** [review]

**Pull request merged**